presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -10,6 +10,26 @@ Linux Operating System
 
 * 64-bit required
 * newer release preferred, especially when running on containers
+* adequate ulimits for the user that the Presto process runs as. Currently, 
+  Presto requires a minimum of 4096 open file descriptors, and strongly 
+  recommends at least 8192 when it runs. Many modern linux systems built to 
+  host large applications such as Presto are expected to support much larger 
+  values, on the order of 32k open files descriptors and above, to ensure that
+  these limits are never hit. These limits are set in 
+  ``/etc/security/limits.conf``:
+
+.. code-block:: none
+
+  presto soft nofile 32768
+  presto hard nofile 65536
+  presto soft nproc 32768
+  presto hard nproc 65536
+
+.. note::
+
+  The root user will often have different ulimits set than do other users.
+  If you see error messages regarding the number of open files, please be 
+  sure that you are changing the values for the presto user and not root.
```

Comment:
Capitalize Presto so it's clear we are saying "the user Presto runs as" and not the `presto` user.

Reply:
changed to "please be sure that you are changing the values for the Presto user and not root."